app/oracle-server/src/main/resources/reference.conf

```diff
@@ -1,8 +1,9 @@
 bitcoin-s {
     oracle {
         rpcport = 9998
+        rpcbind = "0.0.0.0"
```

Comment:
I think this is potentially a problem, we don't want to publish on all interfaces unless this is running inside a docker container. So we probably need some sort of docker specific setting?

Reply:
Yeah this is dangerous, we shouldn't do this

Reply:
It needs to be done inside the container, however we don't want to default it outside of the container.

Reply:
Ok this should be resolved in 4357b6f